src/components/types.ts

```diff
@@ -198,8 +203,8 @@ export interface MultipartFormat extends DataFormat {
 export type MultipartData = MultipartFormat & Updates<MultipartFormat>
 
 export interface BaseInputProps {
-  /** Id of the current user. */
-  sender: string
+  /** Current user. */
```

Comment:
MultimodalInput and TextInput's sender description and type are not changed by BaseInputProps. You need to change the type description in these components' stories